sdk/textanalytics/azure-ai-textanalytics/src/samples/java/com/azure/ai/textanalytics/batch/ExtractKeyPhrasesBatchDocuments.java

```diff
@@ -37,21 +37,33 @@ public static void main(String[] args) {
             new TextDocumentInput("2", "The pitot tube is used to measure airspeed.", "en")
         );
 
+        // Request options: show statistics and model version
         final TextAnalyticsRequestOptions requestOptions = new TextAnalyticsRequestOptions().setShowStatistics(true);
-        final DocumentResultCollection<ExtractKeyPhraseResult> detectedBatchResult = client.extractBatchKeyPhrasesWithResponse(inputs, requestOptions, Context.NONE).getValue();
-        System.out.printf("Model version: %s%n", detectedBatchResult.getModelVersion());
 
-        final TextDocumentBatchStatistics batchStatistics = detectedBatchResult.getStatistics();
+        // Extracting batch key phrases
+        final DocumentResultCollection<ExtractKeyPhraseResult> extractedBatchResult = client.extractBatchKeyPhrasesWithResponse(inputs, requestOptions, Context.NONE).getValue();
```

Comment:
I'd reconsider the WithResponse calls. I thought we added this to encompass all use-cases, but I don't expect usual devs to care about the HTTP response.

Reply:
same to the async case that I mentioned above. Either create another API like `extractBatchKeyPhrases(inputs, requestOptions)` or not use extractBatchKeyPhrases(inputs) as example, I think we should show the user how to use requestOptions so I don't prefer the second way.